tools/operator/testdata/kafka-couchdb-tls-network-spec.yml

```diff
@@ -22,6 +22,8 @@ metrics: true
 gossipEnable: false
 #! enable node ou's in fabric network (true/false)
 enableNodeOUs: true
+#! Provide kafka version
+kafkaVersion: 1.0.2
```

Comment:
I think this should be moved to kafka section as this is specific to kafka instead of declaring here.